src/camunda_worker/tasks/migrations/0003_change_callback_path.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.2 on 2019-07-01 15:01
+
+from django.db import migrations
+
+
+def rewrite_callback_paths(apps, schema_editor):
+    TaskMapping = apps.get_model("tasks.TaskMapping")
+
+    mappings = TaskMapping.objects.filter(
+        callback__regex=r"^camunda_worker\.tasks\.tasks\.\w+$"
+    )
+    for mapping in mappings:
+        bits = mapping.callback.rsplit(".", 1)
+        bits.insert(1, "worker")
+        mapping.callback = ".".join(bits)
+        mapping.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("tasks", "0002_auto_20200120_1240")]
+
+    operations = [
+        migrations.RunPython(rewrite_callback_paths, migrations.RunPython.noop)
```

Comment:
can you correctly implement the reverse migration insead of `noop`, or leave it out so that migrations can not be reverted?

I could now migrate, reverse, and migrate again and then I get `.worker.worker` in the callback.

Reply:
`reverse_code` is removed from operation